pcsx2/gui/FrameForGS.cpp

```diff
@@ -171,6 +182,13 @@ void GSPanel::DoResize()
 	float zoom = g_Conf->GSWindow.Zoom.ToFloat()/100.0;
 	if( zoom == 0 )//auto zoom in untill black-bars are gone (while keeping the aspect ratio).
 		zoom = std::max( (float)arr, (float)(1.0/arr) );
+	if (arr == 1 && EmuConfig.Gamefixes.FMVinSoftwareHack && g_Conf->GSWindow.IsFullscreen) {
+		//avoids crash on some systems switching HW><SW in fullscreen aspect ratio's with FMV Software switch.
+		if (zoom < 1.001)
+			zoom = (g_Conf->GSWindow.Zoom.ToFloat() + 1.0) / 100.0;
+		if (zoom > 0.999)
+			zoom = (g_Conf->GSWindow.Zoom.ToFloat() - 1.0) / 100.0;
```

Comment:
Shouldn't it be
```
if (zoom > 0.999 && zoom < 1.001)
	zoom = (g_Conf->GSWindow.Zoom.ToFloat() - 1.0) / 100.0;
```
? Otherwise if `g_Conf->GSWindow.Zoom.ToFloat()` is 101 you'll end up with a zoom of 1.